### PR TITLE
42 payment model

### PR DESCRIPTION
**Pull request by garrett-vangilder**
## Status

**READY/IN DEVELOPMENT/HOLD**
## Documentation

[ ] I have fully documented features in this PR. Please put an `X` in the box to confirm.
## Migrations

YES 
## Description

A few sentences describing the overall goals of the pull request's commits.

This Pull request updates the payment model to mirror the erd
## Related PRs

List related PRs against other branches:

| branch | PR |
| --- | --- |
| other_pr_production | [link]() |
| other_pr_master | [link]() |
## Resolves Issue Number

number [link]()
## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.

``` sh
git pull --prune
git checkout 42-PaymentModel
rm -rf bangazon.db
rm -rf Migrations/
bangazon migration
bangazon run
```
1. 
## Impacted Areas in Application

List general components of the application that this PR will affect:
- This impacts the User model heavily

### CODE REVIEW

**Comment by garrett-vangilder** [comment]
LGTM

**Comment by garrett-vangilder** [comment]
@TeamCharles/team ready for review whenever you are ready.

**Comment by matthamil** [comment]
:shipit:

File: Models/PaymentType.cs [hunk 1]
```diff
@@ -23,6 +23,6 @@ public class PaymentType
     [StringLength(20)]
     public string AccountNumber { get; set; }
     public int CustomerId {get;set;}
```
Comment (MattKraatz) [inline Models/PaymentType.cs]: this needs to be UserId as well

Comment (MattKraatz) [reply Models/PaymentType.cs]: and UserId should be [Required]

**Review by MattKraatz — COMMENTED** [review]
Couple small changes

**Comment by garrett-vangilder** [comment]
@TeamCharles/team changes made that follow @MattKraatz suggestions, please go ahead and review once again.

**Comment by matthamil** [comment]
:shipit: :+1: LGTM

**Comment by Anulfo** [comment]
LGTM

**Review by Anulfo — APPROVED** [review]
LGTM

**Comment by matthamil** [comment]
LGTM

**Comment by daynewright** [comment]
LGTM

**Comment by garrett-vangilder** [comment]
@MattKraatz  and @daynewright maybe again?

**Comment by MattKraatz** [comment]
LGTM @garrett-vangilder

**Pull request merged**